Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,30 +6,33 @@ class Program
     {
         static void Main(string[] args)
         {
-            //Bmi Calculator
-            Console.WriteLine("BMI Calculator:");
-            
-            //Full Name
+            Bmi1(number:1);
```

Comment:
Bmi1 is a noun. Also 1 next to method doesn't mean anything. Lastly, ``number`` next to :1 is not needed, because it is called a positional stuff and is used when you specify argument out of order, especially when it's optional.

Should be: ``CalculateBmi(1);``